### DISCUSSION TITLE
1.1.0 사용 가능한가요?

### DISCUSSION THREAD

**Discussion by dlaanfrhrldla** [changdoc/naver-vaccine-macro#133] [Q&A]
안녕하세요 개발자님, 덕분에 잔여백신도 일찍 구했고, 가족분들께도 많이 도움 드리고 있습니다.
질문 하나 드리고 싶은데, 혹시 새로 업데이트 중이신 카카오맵 연동 1.1.0 버젼은 언제부터 사용이 가능할까요?
감사합니다.

**Comment by changdoc** [comment]
카카오맵쪽 사용은 사용 방법이 조금 복잡하고 백신 종류 선택하는 부분이 아직 미완성이라 가이드를 따로 작성하고 있지 않았네요.
작업 진행 되는대로 신규 버전으로 릴리즈 해두겠습니다.

**Reply by changdoc** [reply]
네이버맵쪽에서 갱신 하는 부분 동작은 오늘 확인하긴 했었는데... 예약 신청까지 정상적으로 하는지 확실하게 검증이 안되어서 아직 공개를 못하고 있네요.

graphql response를 매크로 창에서 저장하고 그 정보와 새로고침중인 화면상의 병원들이 일치하게 해두고
맵에 있는 매크로 동작시키면 맵상에서 잔여 백신이 발생했다고 판단될때 예약 신청 페이지로 이동하여 신청 까지 수행하는데요.

좀 더 테스트 및 보완 후에 정식 릴리즈 하려고 생각 중입니다.
